doc/plugin_agent_workloadattestor_k8s.md

```diff
@@ -4,9 +4,35 @@ The `k8s` plugin generates kubernetes-based selectors for workloads calling the
 It does so by retrieving the workload's pod ID from its cgroup membership, then querying
 the kubelet for information about the pod.
 
+The plugin can talk to the kubelet via the insecure read-only port or the
+secure port. Both X509 client authentication and bearer token (e.g. service
+account token) authentication to the secure port is supported.
+
+Verifying the certificate presented by the kubelet over the secure port is
+optional. The default is to verify, based on the certificate file passed via
+`kubelet_ca_path`. `skip_kubelet_verification` can be set to disable
+verification.
+
+**Note** kubelet authentication via bearer token requires that the kubelet be
+started with the the `--authentication-token-webhook` flag. See [Kubelet authentication/authorization](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-authentication-authorization/)
+for details.
+
+**Note** The kubelet uses the TokenReview API to validate bearer tokens. This
+requires reachability to the Kubernetes API server. Therefore API server downtime can
+interrupt workload attestation. The `--authentication-token-webhook-cache-ttl` kubelet flag
+controls how long the kubelet caches TokenReview responses and may help to
+mitigate this issue. A large cache ttl value is not recommended however, as
+that can impact permission revocation.
+
 | Configuration | Description |
 | ------------- | ----------- |
-| kubelet_read_only_port | The port on which the kubelet has exposed its read-only API. |
+| `kubelet_read_only_port` | The kubelet read-only port. This is mutually exlusive with `kubelet_secure_port`. |
+| `kubelet_secure_port` | The kubelet secure port. It defaults to `10250` unless `kubelet_read_only_port` is set. |
+| `kubelet_ca_path` | The path on disk to a file containing CA certificates necessary to verify the kubelet certificate. Required unless `skip_kubelet_verification` is set. |
```

Comment:
I think this should just default to /run/secrets/kubernetes.io/serviceaccount/ca.crt

Reply:
I saw that in your PR and it seems like a good choice. The kubelet certificate in my minikube environment was not signed by a cert in that bundle however, which is why I hesitated making it the default. From what I can tell from the [docs](https://kubernetes.io/docs/tasks/tls/managing-tls-in-a-cluster/), `ca.crt` should contain the trust bundle for the cluster so I can't imagine why it isn't working in minikube. Either way, if we're confident that isn't the case in normal deployments, I will make it the default.

Reply:
Ok. I did not try on minikube, but I am running this PR on an Amazon EKS cluster now and it's working with ca.crt